bindings/rust/src/pippenger.rs

```diff
@@ -72,43 +73,49 @@ macro_rules! pippenger_mult_impl {
             pub fn from(points: &[$point]) -> Self {
                 let npoints = points.len();
                 let mut ret = Self {
-                    points: Vec::with_capacity(npoints),
+                    points: vec![Default::default(); npoints],
```

Comment:
The intent here is to avoid a useless zero initialization when ret.points will be initialized less than 10 lines below.